eth_worker/eth_src/celery_app.py

```diff
@@ -64,9 +67,10 @@
 app.conf.beat_schedule = {
     'third-party-transaction-sync': {
         'task': celery_utils.eth_endpoint('synchronize_third_party_transactions'),
-        'schedule': 30, # Every 30 seconds
+        'schedule': config.THIRD_PARTY_SYNC_CHECK_PERIOD_SECONDS, # Every 5 seconds by default
```

Comment:
@enjeyw This might be too frequent-- The action often takes over 30s to do, so the logs already have a lot of `Skipping execution of synchronizing filter 1, as it is already running in another process`. The endpoint is pretty quick to rate-limit an IP, so hitting it too hard will bog down transactions as well.

Reply:
ohhh interesting. I was looking for a default of 5 seconds because that's the typical block time on many ethereum chains, and i noticed that the synchronisation was currently getting multiple blocks per check. Is there an easy way to get the action time down? what's the bottleneck?

Reply:
I don't think there is any way to speed it up, since the bottleneck is the actual `filter.get_all_entries` call. It might just be slow on sokol, it might well run faster in prod (would be easy to check this in the logs)

Reply:
from memory we actually found (and cleared) a different bottleneck for this in the last few months, though i can't recall what it is....